examples/c/ex_thread.c

```diff
@@ -86,8 +86,10 @@ main(int argc, char *argv[])
     error_check(cursor->insert(cursor));
     error_check(session->close(session, NULL));
 
-    for (i = 0; i < NUM_THREADS; i++)
+    for (i = 0; i < NUM_THREADS; i++) {
+        threads[i].name_index = 0;
```

Comment:
I wonder if we should initialize the name_index whenever we call `__wt_thread_create` instead. I'm going to defer to @jeremythorp  to answer this.

Reply:
I think it makes sense to initialise these values in this (main) function as that is where they are declared.

Reply:
So I think @bzm-jas 's code here is ok as it is.